x-pack/plugins/cases/public/components/owner_context/use_owner_context.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useContext } from 'react';
+import { OwnerContext } from '.';
+
+export const useOwnerContext = () => {
+  const ownerContext = useContext(OwnerContext);
+
+  if (ownerContext.length === 0) {
```

Comment:
Can ownerContext ever be undefined? Doesn't look like it, but just wanted to make sure

Reply:
No, because is initialized as an empty array.